.gitmodules

```diff
@@ -1,3 +1,7 @@
 [submodule "json"]
 	path = json
 	url = https://github.com/nlohmann/json.git
+[submodule "example/Blinky/mbed-os"]
```

Comment:
Not a big fan of adding a git submodule for mbed targets. This would imply that we should add a new dependency for EVERY different target...